gobblin-runtime/src/main/java/org/apache/gobblin/runtime/mapreduce/GobblinWorkUnitsInputFormat.java

```diff
@@ -203,7 +210,11 @@ public Text getCurrentValue()
     @Override
     public float getProgress()
         throws IOException, InterruptedException {
-      return (float) this.currentIdx / (float) this.totalPaths;
+      if (MRJobLauncher.isCustomizedProgressReportEnabled(properties)) {
+        return 0.0f;
```

Comment:
Can we make the progress of finishing deserialize wu configurable, and return 
(float) this.currentIdx / (float) this.totalPaths * progress_of_finishing_deserialize here

Reply:
Will address